internal/gatewayapi/testdata/httproute-attaching-to-listener-with-backend-backendref-mixed-address-type.out.yaml

```diff
@@ -188,9 +189,9 @@ httpRoutes:
         status: "True"
         type: Accepted
       - lastTransitionTime: null
-        message: mixed FQDN and IP or Unix address type for the same backendRef is
-          not supported
-        reason: ResolvedRefs
```

Comment:
According to Gateway API, `ResolvedRefs` reason should only be used when the condidtion is`True`.

```
	// This reason is used with the "ResolvedRefs" condition when the condition
	// is true.
	RouteReasonResolvedRefs RouteConditionReason = "ResolvedRefs"
```